workers/unity/Assets/Gdk/Core/Collections/BlittableList.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Assets.Gdk.Core.Collections
+{
+    public struct BlittableList<T> : IList<T>
+    {
+        private readonly uint handle;
+
+        internal List<T> Internal => ListProvider<T>.Get(handle);
+
+        internal BlittableList(uint handle)
+        {
+            this.handle = handle;
+        }
+
+        public static implicit operator List<T>(BlittableList<T> bl) => bl.Internal;
+
+        #region IList<T>
+        public T this[int index] {
+            get { return Internal[index]; }
+            set { Internal[index] = value; }
+        }
+
+        public int Count => Internal.Count;
+
+        public bool IsReadOnly => ((ICollection<T>)Internal).IsReadOnly;
+
+        public void Add(T item) => Internal.Add(item);
```

Comment:
If you want this to be like a list then you should probably cover all the public methods of List. Currently it will only act like List as much as IList.